test/core/iomgr/BUILD

```diff
@@ -81,6 +81,7 @@ grpc_cc_test(
         "//:grpc",
         "//test/core/util:grpc_test_util",
     ],
+    tags = ["exclude_windows"],
```

Comment:
nit:
e.g. tensorflow is using the tag name "no_windows" instead - maybe that's better naming for the tag? https://github.com/tensorflow/tensorflow/blob/ec2dc801e89547ef854b26bc7fe3f8c1adfad87a/tensorflow/compiler/tf2tensorrt/BUILD#L34

"no_windows" also seems to be used by the bazel team itself, so it feels to be a more standard choice than "exclude_windows":
https://github.com/bazelbuild/bazel/blob/677ba95e881707b121ce6588883b3ab68ed4c533/src/test/shell/bazel/BUILD#L751

I also noticed that we could define a bazel testsuite for windows tests, which would allow us to specify the tags to exclude directly - that's good to know:
https://github.com/bazelbuild/bazel/blob/677ba95e881707b121ce6588883b3ab68ed4c533/src/test/shell/bazel/BUILD#L759

Reply:
Tag name changed per your suggestion
windows_tests applies mostly inside a single BUILD file. In this case, there are multiple BUILD files and thousands of targets, most of them runnable on Windows. AFAIK there's no good global filter for "bazel test /..." except for test_tag_filters in the bazelrc file